homeassistant/components/statistics/sensor.py

```diff
@@ -170,15 +177,27 @@ def valid_state_characteristic_configuration(config: dict[str, Any]) -> dict[str
     return config
 
 
+def valid_boundary_configuration(config: dict[str, Any]) -> dict[str, Any]:
+    """Validate that either sampling_size or max_age are provided."""
+
+    if config.get(CONF_SAMPLES_MAX_BUFFER_SIZE) is None:
+        config[CONF_SAMPLES_MAX_BUFFER_SIZE] = DEFAULT_BUFFER_SIZE
+        if config.get(CONF_MAX_AGE) is None:
+            _LOGGER.warning(
+                DEPRECATION_WARNING_SIZE,
+                str(DEFAULT_BUFFER_SIZE),
```

Comment:
There's no need to stringify non-string format arguments
```suggestion
                DEFAULT_BUFFER_SIZE,
```

Reply:
I wonder if I did this for some linter reason... done